tracker_node:
  ros__parameters:
    sample_rate: 15.0
    delta_max: 20.0
    max_update_pause: 6000.0 
    estimatation_uncertainty: [100.,100.,100.,100.,100.,100.]
    measurment_uncertainty: [0.25,0.25]
    processnoise: [.9, .9, 0.6, 0.6, 0.1, 0.1]

visualizer_node:
  ros__parameters:
    delta_max: 20.0
    publish_visualization: false
    draw_visualization: true
